### PR TITLE
Add more description to main readme and multi-node readme

### PR DESCRIPTION
**Pull request by andrecolin**
Add more description to readme for main and multi node to install golang 1.7.x and setting kubeconf for multinode vagrant install to setup kubeconf

### CODE REVIEW

File: hack/multi-node/README.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ This will generate the default assets in the `cluster` directory and launch mult
 ./bootkube-up
 ```
 
+Configure Kubeconfig env variable allow using kubectl without paramaters
+
+```
+export KUBECONFIG=$GOPATH/src/github.com/kubernetes-incubator/bootkube/hack/multi-node/cluster/auth/kubeconfig
+```
+
+Verify kubctl
```
Comment (yifan-gu) [inline hack/multi-node/README.md]: s/kubctl/kubectl

File: README.md [hunk 2]
```diff
@@ -89,6 +89,28 @@ go get -u github.com/kubernetes-incubator/bootkube
 cd $GOPATH/src/github.com/kubernetes-incubator/bootkube
 ```
 
+Install Golang 1.7.x, seems to be issues with 1.6.3 on ubuntu 16.04
```
Comment (yifan-gu) [inline README.md]: Actually in the CI we are building with 1.8.3 now.

**Comment by yifan-gu** [comment]
@andrecolin Thanks for the PR. Maybe instead of 1.7.x, we can go straight with 1.8.x? 
Btw, have you signed CLA?

File: hack/multi-node/README.md [hunk 3]
```diff
@@ -10,6 +10,22 @@ This will generate the default assets in the `cluster` directory and launch mult
 ./bootkube-up
 ```
 
+Configure Kubeconfig env variable allow using kubectl without paramaters
```
Comment (diegs) [inline hack/multi-node/README.md]: nit 1: I would add `(optional)` to this step. It is not strictly required.
nit 2: "... env variable to allow"
nit 3: s/paramaters/parameters

**Comment by dghubble** [comment]
This is rather outdated and didn't address the original review. Going to close, but feel free to open if you'd like to take on responsibility for it.

**Pull request closed without merging**